lib/mpl_toolkits/basemap/pyproj.py

```diff
@@ -1,868 +1,4 @@
-"""
-Cython wrapper to provide python interfaces to
-PROJ.4 (http://trac.osgeo.org/proj/) functions.
+# pyproj compatibility layer for externally hosted package
+# pyproj development occurs at https://github.com/jswhit/pyproj
```

Comment:
This file probably needs to be eliminated entirely. It is getting picked up by the pyproj imports the other files are doing because it is in the same directory.